oops/backplane/where.py

```diff
@@ -205,3 +204,114 @@ def where_all(self, *backplane_keys):
 Backplane._define_backplane_names(globals().copy())
 
 ################################################################################
+
+
+################################################################################
+# UNIT TESTS
+################################################################################
+
+import unittest
+from oops.meshgrid                      import Meshgrid
+from oops.unittester_support            import TESTDATA_PARENT_DIRECTORY
+from oops.backplane.unittester_support  import show_info
+
+#===========================================================================
+def exercise(bp, printing, saving, dir, refdir,
+             planet=None, moon=None, ring=None,
+             undersample=16, use_inventory=False, inventory_border=2):
+    """generic unit tests for where.py"""
+
+    if planet is not None:
+        test = bp.where_intercepted(planet)
+        show_info(bp, 'Mask of planet intercepted', test,
```

Comment:
By the way, because you are using the same last four arguments in every call to show_info, I suggest you create a dictionary called "options", with these four items, and then change the call to show_info(..., **options). That will also make it easier to add some new option in the future if we decide we need it.